platform/ios/src/MGLMapView.mm

```diff
@@ -1216,8 +1219,17 @@ - (void)handlePanGesture:(UIPanGestureRecognizer *)pan
     else if (pan.state == UIGestureRecognizerStateChanged)
     {
         CGPoint delta = [pan translationInView:pan.view];
-        _mbglMap->moveBy({ delta.x, delta.y });
-        [pan setTranslation:CGPointZero inView:pan.view];
+
+        toCamera = [self currentCameraWithEstimatedEndPoint:delta panGesture:pan];
+        
+        if ([self.delegate respondsToSelector:@selector(mapView:shouldChangeFromCamera:toCamera:)]
+            && ![self.delegate mapView:self shouldChangeFromCamera:oldCamera toCamera:toCamera])
+        {
+            self.camera = oldCamera;
```

Comment:
OK, let’s remove this line and similar lines, then. Setting the camera also cancels any pending animations, which could be unexpected.